src/start/CompatibilityScreen.js

```diff
@@ -15,17 +17,58 @@ const styles = StyleSheet.create({
     color: 'white',
     fontSize: 20,
     margin: 8,
-  }
+  },
+  touchTarget: {
+    flexDirection: 'row',
+    alignSelf: 'stretch',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  logo: {
+    width: 200,
+    height: 100,
+  },
 });
 
-export default class CompatibilityScreen extends React.Component {
 
+export default class CompatibilityScreen extends React.Component {
+  handleClick = (link: string) => {
+    Linking.canOpenURL(link).then(supported => {
+      if (!supported) {
+        console.log('Can\'t open URL: ', link);
+      } else {
+        Linking.openURL(link);
+      }
+    }).catch(err => console.log('An error occurred ', err));
+  }
   render() {
     return (
       <View style={styles.screen}>
         <Logo />
-        <Text style={styles.text}>This app is too old!</Text>
-        <Text style={styles.text}>Please download the latest version from the App Store.</Text>
+        <Text style={styles.text}>This is an outdated version of Zulip.</Text>
+        {(Platform.OS === 'ios') ? (
```

Comment:
You can create a component called AppStoreLink and have two versions, one in `AppStoreLink.ios.js` and another in `AppStoreLink.android.js`. Then you just import it like `import AppStoreLink from './AppStoreLink'` and you don't have to worry about the platform check at all :)

React Native will automatically import the correct version for the platform